src/main/io/gps.c

```diff
@@ -154,6 +277,8 @@ void gpsProcessNewSolutionData(void)
     // Set sensor as ready and available
     sensorsSet(SENSOR_GPS);
 
+	updateEstimatedGPSFix();
```

Comment:
Should be conditional on `allow_gps_fix_estimation ` setting or `canEstimateGPSFix `being true surely ? No point calling this if use option not set, isn't a plane, has no mag etc.